testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -962,9 +962,9 @@ def ref_PBC(self):
                 dtype=np.float32),
             'BSph': (47.923367, np.array([26.82960892, 31.5592289, 30.98238945], dtype=np.float32)),
             'PAxes': np.array([
-                [-0.85911708, 0.19258726, 0.4741603],
-                [-0.07520116, -0.96394227, 0.25526473],
-                [-0.50622389, -0.18364489, -0.84262206]])
+                [ 0.85911708, -0.19258726, -0.4741603 ],
```

Comment:
Again, 

Updated the test PAxes to an array that returns 1. The original returned -1.